#exec addr
#coins = "1GaHYpWmqAJsqRwrpoNcB8VvgKtSwjcHqt"
#token = "12hpJBHybh1mSyCijQ2MQJPk7z7kZ7jnQa"
#trade = "1BXvgjmBw1aBgmGn1hjfGyRkmN3krWpFP4"
#ticket = "16htvcBNSEA7fZhAdLJphDwQRQJaHpyHTp"
#privacy = "1FeyE6VDZ4FYgpK1n2okWMDAtPkwBuooQd"


[[TransferCase]]
id = "btyTrans1"
command = "send bty transfer -a 10 -t 1D9xKRnLvV2zMtSxSx33ow1GF4pcbLcNRt -k 12qyocayNF7Lv6C9qW4avxs2E7U41fKSfv"
from = "12qyocayNF7Lv6C9qW4avxs2E7U41fKSfv"
to = "1D9xKRnLvV2zMtSxSx33ow1GF4pcbLcNRt"
amount = "10"
checkItem = ["balance"]
repeat = 1


[[TransferCase]]
id = "btyTrans2"
command = "send bty transfer -a 1 -t 17UZr5eJVxDRW1gs7rausQwaSUPtvcpxGT -k 1D9xKRnLvV2zMtSxSx33ow1GF4pcbLcNRt"
from = "1D9xKRnLvV2zMtSxSx33ow1GF4pcbLcNRt"
to = "17UZr5eJVxDRW1gs7rausQwaSUPtvcpxGT"
amount = "1"
checkItem = ["balance"]
repeat = 5
dep = ["btyTrans1"]


[[TransferCase]]
id = "failBtyTrans"
command = "send bty transfer -a 15 -t 1D9xKRnLvV2zMtSxSx33ow1GF4pcbLcNRt -k 17UZr5eJVxDRW1gs7rausQwaSUPtvcpxGT"
from = "17UZr5eJVxDRW1gs7rausQwaSUPtvcpxGT"
to = "1D9xKRnLvV2zMtSxSx33ow1GF4pcbLcNRt"
amount = "15"
checkItem = ["balance"]
fail = true


[[TransferCase]]
id = "btyTrans3"
command = "send bty transfer -a 1 -t 12hpJBHybh1mSyCijQ2MQJPk7z7kZ7jnQa -k 17UZr5eJVxDRW1gs7rausQwaSUPtvcpxGT"
from = "17UZr5eJVxDRW1gs7rausQwaSUPtvcpxGT"
to = "12hpJBHybh1mSyCijQ2MQJPk7z7kZ7jnQa"
amount = "1"
checkItem = ["balance"]
dep = ["btyTrans2"]


[[WithdrawCase]]
id = "btyWithdraw"
command = "send bty withdraw -a 0.1 -e token -k 17UZr5eJVxDRW1gs7rausQwaSUPtvcpxGT"
addr = "17UZr5eJVxDRW1gs7rausQwaSUPtvcpxGT"
amount = "0.1"
checkItem = ["balance"]
repeat = 5
dep = ["btyTrans3"]


[[WithdrawCase]]
id = "failWithdraw"     #带有fail前缀表示用例本身是失败的，在autotest程序中集成了这个特性
command = "send bty withdraw -a 1.1 -e token -k 17UZr5eJVxDRW1gs7rausQwaSUPtvcpxGT"
addr = "17UZr5eJVxDRW1gs7rausQwaSUPtvcpxGT"
amount = "1.1"
checkItem = ["balance"]
fail = true

